charters/DataOperations/charter.md

```diff
@@ -0,0 +1,99 @@
+
+## Definitions
+
+*Data contribution*: a set of raw data derived from the same experimental process, and the metadata describing that process
+
+*Release* (verb): to make metadata and data openly accessible
+
+*Data distribution*: the packaging of metadata and data from multiple data contributions in meaningful ways, along with relevant analysis pipelines, possible integrative analysis results, user vignettes, access features, and any announcements or documentation supporting it
+
+*HCA Snapshot*: a type of data distribution that occurs on a regular cadence containing all HCA data currently available that meets predetermined criteria, such as quality standards
+
+## Objectives
+
+The objective of the Data Operations is to connect the scientific community to tools and resources from the DCP which maximize the value and use of HCA data such as data, metadata, analysis tools, and data access features. Data Operations aims to maintain a reliable and efficient data workflow and shepherds data through the workflow.
+
+## In-scope
+
+- Data operations
+  - Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data
+  - Coordinate DCP-wide end-to-end testing before being released and continued QC checks after being released to ensure data and data resource integrity and useability through component updates
+  - Identify data workflow- and data distribution-associated engineering needs of all DCP components and participate in crafting technical specifications
+  - Establish and integrate into the DCP roadmap a reproducible plan that includes the specifications for and the timing of iterative data distribution to the community
+  - Monitor the status of data contributions at each step in the DCP data workflow from initial identification of suitability and shepherd all the way through release and potential inclusion on tertiary portals and/or in additional data distribution
+  - Identify processes in the data workflow that require improved monitoring, streamlined troubleshooting, automation, etc. towards a more scalable and efficient workflow with increased data contribution bandwidth
+  - Engage with Governance Group and Analysis Work Group to identify data quality standards which dictate data eligible to be released and data eligible to be included in data distributions
+  - Ensure proper checks of data quality standards are put in place within the data workflow
```

Comment:
It would be useful to add "data workflow" to the definitions section. The analysis team normally uses the term "workflow" to refer to just the analysis workflow but I'm not sure that is what's meant in this context.

Reply:
Great idea! Done in a86aa9a

Reply:
I tend to think of this as quality assurance.
How I see these words are "quality assurance" is making sure the data is validated on ingest (not corrupt, not empty, correct schema/format) and making sure everything we say we will do happens and making sure when it happens it is done correctly.
While quality control are metrics that are used to measure the scientific quality of the data (like alignment metrics), which make sense for curated distribution. It has been communicated QC would not be a barrier to data release (access). That being said, the project did include in the original designs file listings or bucketing, to comment on QC of files but the files would still be released (accessible). Happy to talk more.

Reply:
@TimothyTickle I think we're on the same page here, as I have not heard of any potential QC standards that would block _releasing_ data, but possibly some would be in place for inclusion in a data distribution. So we see ourselves ensuring both QA and QC checks, whatever they may be, within the data lifecycle.

Reply:
Maybe language use can help here. The data processing pipelines team, the metadata team and the ingest support team all play roles in QA and QC of data in the platform. Maybe making sure these statements are more inclusive to the other components who also share this responsibility

For example the ingest support charter has this statement

*Working with the User Experience (UX) team to collect and provide feedback from data contributors to the Ingestion Service team about how the infrastructure can be improved*

and the metadata-schema team has 

*Working with Ingestion Service to specify the entity structure for HCA metadata schema*

Making this statements inclusive and about working with other components to deliver this rather than being readable in a way that it suggests that the data operations team is the only group responsible (I know that isn't your intent) might be better

Reply:
Excellent suggestion. I have included more appropriate language for where we will be engaging and supporting other components. b54d1a8

Reply:
This has been quiet, so I will resolve